test/suite/test_verify_disagg.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import os, wiredtiger, wttest
+from helper_disagg import DisaggConfigMixin, disagg_test_class, gen_disagg_storages
+from wtscenario import make_scenarios
+
+# test_verify_disagg.py
+#    SESSION::verify() testing for disagg storage
+
+# FIXME-WT-15047: Implement tests for populated ingest tables verification
+#    (we already have an OpLog imitation in some tests for layered tables)
+
+@disagg_test_class
+class test_verify_disagg(wttest.WiredTigerTestCase, DisaggConfigMixin):
+    hs = [
+        ('empty', dict(fill_hs=False)),
+        ('populated', dict(fill_hs=True)),
+    ]
+    disagg_storages = gen_disagg_storages('test_verify_disagg', disagg_only = True)
+    scenarios = make_scenarios(hs, disagg_storages)
+
+    nitems = 10000
+    timestamp = 2
+
+    conn_base_config = 'disaggregated=(page_log=palm),'
+    conn_config = conn_base_config + 'disaggregated=(role="leader")'
+    conn_config_follower = conn_base_config + 'disaggregated=(role="follower")'
+
+    table_cfg = 'key_format=S,value_format=S,block_manager=disagg,log=(enabled=false)'
+
+    session_follow = None
+    conn_follow = None
+
+    uri = 'layered:test_verify_disagg'
+
+    # Load the page log extension, which has object storage support
+    def conn_extensions(self, extlist):
+        if os.name == 'nt':
+            extlist.skip_if_missing = True
+        DisaggConfigMixin.conn_extensions(self, extlist)
```

Comment:
Looks like it exists inside the `disagg_test_class` class defined in `helper_disagg.py`. I wonder if this is needed or if we inherit from it automatically